core/src/test/java/org/neo4j/ogm/domain/friendships/Person.java

```diff
@@ -26,7 +26,7 @@
     private Long id;
     private String name;
 
-    @Relationship(type = "FRIEND_OF")
+    @Relationship(type = "FRIEND_OF", direction = Relationship.UNDIRECTED)
```

Comment:
If you read the test it kind of makes no sense without some sort of direction (when wanting to load the change from the other side of a relationship). It technically should be incoming or outgoing but since this test "intended" to have no direction, i've marked it as such. Not sure how it worked before that tbh.